From b880dc016cfde814ad6257f6b8fe90fbe3ca167c Mon Sep 17 00:00:00 2001
From: Janne Grunau <j@jannau.net>
Date: Sun, 17 Mar 2024 17:12:24 +0100
Subject: [PATCH 1/1] configs: apple: Do not show the boot menu automatically

It is empty and automatic boot does not work.

This is a defensive commit against the fedora package which caries a
patch to default CONFIG_AUTOBOOT_MENU_SHOW to y.

Signed-off-by: Janne Grunau <j@jannau.net>
---
 configs/apple_m1_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/apple_m1_defconfig b/configs/apple_m1_defconfig
index aceb8c06775..54608bf123f 100644
--- a/configs/apple_m1_defconfig
+++ b/configs/apple_m1_defconfig
@@ -4,6 +4,7 @@ CONFIG_ARCH_APPLE=y
 CONFIG_DEFAULT_DEVICE_TREE="apple/t8103-j274"
 CONFIG_SYS_BOOTM_LEN=0x800000
 CONFIG_SYS_LOAD_ADDR=0x0
+# CONFIG_AUTOBOOT_MENU_SHOW is not set
 CONFIG_BOOTCOMMAND="bootflow scan -b"
 CONFIG_USE_PREBOOT=y
 CONFIG_SYS_CBSIZE=256
